app/src/main/java/org/jellyfin/androidtv/util/profile/MediaCodecCapabilitiesTest.kt

```diff
@@ -33,6 +41,48 @@ class MediaCodecCapabilitiesTest {
 		CodecProfileLevel.AVCLevel4
 	)
 
+	private fun getHevcLevelString(profile: Int) : String {
+		val level = getDecoderLevel(MediaFormat.MIMETYPE_VIDEO_HEVC, profile)
+
+		when {
+			level < CodecProfileLevel.HEVCMainTierLevel1 -> return "0"
+			level < CodecProfileLevel.HEVCMainTierLevel2 -> return "30"
+			level < CodecProfileLevel.HEVCMainTierLevel21 -> return "60"
+			level < CodecProfileLevel.HEVCMainTierLevel3 -> return "63"
+			level < CodecProfileLevel.HEVCMainTierLevel31 -> return "90"
+			level < CodecProfileLevel.HEVCMainTierLevel4 -> return "93"
+			level < CodecProfileLevel.HEVCMainTierLevel41 -> return "120"
+			level < CodecProfileLevel.HEVCMainTierLevel5 -> return "123"
+			level < CodecProfileLevel.HEVCMainTierLevel51 -> return "150"
+			level < CodecProfileLevel.HEVCMainTierLevel52 -> return "153"
+			level < CodecProfileLevel.HEVCMainTierLevel6 -> return "156"
+			level < CodecProfileLevel.HEVCMainTierLevel61 -> return "180"
+			level < CodecProfileLevel.HEVCMainTierLevel62 -> return "183"
+			else -> return "186"
```

Comment:
Where do these numbers come from? It might be better if we add constants for them.

Reply:
These are the numbers ffprobe uses and therefore the Jellyfin server. Each number is the level to one decimal place, with the decimal point removed, then multiplied by 3. So level 4.1 becomes 41*3=123 and level 5 becomes 50*3=150.

I'll add this explanation to a comment and make them constants too.

Reply:
That would be really helpful!